Fix schema errors - existing defects not detected by CI

### <!-- Component -->

<!-- C8-API- -->

### Description

Discovered via: https://github.com/camunda/camunda/issues/46272

These specification defects have been merged to main while CI validation was silently failing.

decision-instances.yaml
```
 375:17    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionInputItem.properties.inputId
 377:19    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionInputItem.properties.inputName
 379:20    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionInputItem.properties.inputValue
 386:18    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionOutputItem.properties.outputId
 388:20    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionOutputItem.properties.outputName
 390:21    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionOutputItem.properties.outputValue
 392:16    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionOutputItem.properties.ruleId
 394:19    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionOutputItem.properties.ruleIndex
```

deployments.yaml
```
 292:32    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentDecisionRequirementsResult.properties.decisionRequirementsId
 294:34    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentDecisionRequirementsResult.properties.decisionRequirementsName
 296:17    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentDecisionRequirementsResult.properties.version
 299:22    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentDecisionRequirementsResult.properties.resourceName
 323:17    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentFormResult.properties.version
 326:22    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentFormResult.properties.resourceName
 341:20    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentResourceResult.properties.resourceId
 343:22    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentResourceResult.properties.resourceName
 345:17    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentResourceResult.properties.version
```

incidents.yaml
```
 406:19    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.IncidentResult.properties.errorType
 416:22    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.IncidentResult.properties.creationTime
 419:15    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.IncidentResult.properties.state
```

jobs.yaml
```
 409:28    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.ActivatedJobResult.properties.elementInstanceKey
```

process-definitions.yaml
```
 463:14    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.ProcessDefinitionMessageSubscriptionStatisticsQuery.properties.page
 509:14    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.ProcessDefinitionInstanceStatisticsQuery.properties.page
```

process-instances.yaml
```
 1193:11    error  required-properties-must-exist         Every entry in `required` must match a key in `properties`.                                                                            components.schemas.ProcessInstanceResult.required[9]
 1203:34    error  require-property-descriptions          Schema properties must have a description.                                                                                             components.schemas.ProcessInstanceResult.properties.processDefinitionVersion
 1209:19    error  require-property-descriptions          Schema properties must have a description.                                                                                             components.schemas.ProcessInstanceResult.properties.startDate
 1212:17    error  require-property-descriptions          Schema properties must have a description.                                                                                             components.schemas.ProcessInstanceResult.properties.endDate
 1415:36  warning  schema-key-properties-must-be-strings  Make the `...Key` property a `string`.                                                                                                 components.schemas.ProcessInstanceModificationActivateInstruction.properties.ancestorElementInstanceKey
 1548:36  warning  schema-key-properties-must-be-strings  Make the `...Key` property a `string`.                                                                                                 components.schemas.DirectAncestorKeyInstruction.properties.ancestorElementInstanceKey
```

user-tasks.yaml
```
 457:34    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.UserTaskFilter.properties.processInstanceVariables
 461:24    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.UserTaskFilter.properties.localVariables
```

### Motivation

These are defects that flew under the radar while our spec validation in CI was silently failing.

### Acceptance criteria

- [x] Fixes pass validation: `npx @stoplight/spectral lint zeebe/gateway-protocol/src/main/proto/v2/rest-api.yaml --ruleset zeebe/gateway-protocol/.spectral.yaml --fail-severity error --quiet` _*and*_ `npx @stoplight/spectral lint zeebe/gateway-protocol/src/main/proto/v2/*.yaml --ruleset zeebe/gateway-protocol/.spectral.yaml --fail-severity error --quiet`


### Links

_No response_

### Breakdown

_No response_

I am patching these specifications defects in order to get specification validation to pass. This is necessary to have a green baseline for automated quality control for an urgent fix to unblock QA in #46271.

I am tagging the authors of the schemas below to request review of the patched content (this review does not block the patch). If any further patching is required, please raise a separate PR. 

My apologies for this, our automated validation in CI failed silently and these didn't get caught in the PR quality gate. 

If you are tagged in this comment, please review the entire comment as there may be multiple schemas that you wrote, that I have patched.

@nicpuppa You are the author of this schema. Please review the content and raise another PR to patch it if it needs it. 

decision-instances.yaml
```
 375:17    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionInputItem.properties.inputId
 377:19    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionInputItem.properties.inputName
 379:20    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionInputItem.properties.inputValue
 386:18    error  require-property-descriptions  Schema properties must have a description. 
```

@nicpuppa this one is yours too.
```                                                                                            components.schemas.EvaluatedDecisionOutputItem.properties.outputId
 388:20    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionOutputItem.properties.outputName
 390:21    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionOutputItem.properties.outputValue
 392:16    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionOutputItem.properties.ruleId
 394:19    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.EvaluatedDecisionOutputItem.properties.ruleIndex
```

@nicpuppa you were the last one to touch this, and @danielkelemen  you are the original author. Please review the patched content and raise a PR if it needs further patching.

deployments.yaml
```
 292:32    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentDecisionRequirementsResult.properties.decisionRequirementsId
 294:34    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentDecisionRequirementsResult.properties.decisionRequirementsName
 296:17    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentDecisionRequirementsResult.properties.version
 299:22    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentDecisionRequirementsResult.properties.resourceName
```

@danielkelemen  you are the original author of this schema. Please review the patched content and raise a PR if it needs further patching.

deployments.yaml
```
 323:17    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentFormResult.properties.version
 326:22    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentFormResult.properties.resourceName
 341:20    error  require-property-descriptions  Schema properties must have a description. 
```

@kzraitel you are the original author of this schema. Please review the patched content and raise a PR if it needs further patching.

deployments.yaml
```                                                                                            components.schemas.DeploymentResourceResult.properties.resourceId
 343:22    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentResourceResult.properties.resourceName
 345:17    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.DeploymentResourceResult.properties.version
```

@mathias-vandaele you are the author of this schema. Please review the patched content and raise a PR if it needs further patching.

It specified that `value` is required, but there is no `value` field. I have removed it from the `required` array. Minimal fix. Is the schema correct? Should there be a `value` field in the schema?

cluster-variables.yaml
```
 394:11    error  required-properties-must-exist  Every entry in `required` must match a key in `properties`.                                                                            components.schemas.ClusterVariableResultBase.required[1]
```

@danielkelemen you are the original author of this schema. Please review the patched content and raise a PR if it needs further patching.

incidents.yaml
```
 406:19    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.IncidentResult.properties.errorType
 416:22    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.IncidentResult.properties.creationTime
 419:15    error  require-property-descriptions  Schema properties must have a description.                                                                                             components.schemas.IncidentResult.properties.state
```

@romansmirnov you are the original author of this schema. Please review the patched content and raise a PR if it needs further patching.

process-instances.yaml
```
 1193:11    error  required-properties-must-exist         Every entry in `required` must match a key in `properties`.                                                                            components.schemas.ProcessInstanceResult.required[9]
 1203:34    error  require-property-descriptions          Schema properties must have a description.                                                                                             components.schemas.ProcessInstanceResult.properties.processDefinitionVersion
 1209:19    error  require-property-descriptions          Schema properties must have a description.                                                                                             components.schemas.ProcessInstanceResult.properties.startDate
 1212:17    error  require-property-descriptions          Schema properties must have a description.                                                                                             components.schemas.ProcessInstanceResult.properties.endDate
```

--- a/zeebe/gateway-protocol/src/main/proto/v2/audit-logs.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/audit-logs.yaml
@@ -72,6 +72,8 @@ components:
     AuditLogResult:
       description: Audit log item.
       type: object
+      required:
+        - rootProcessInstanceKey
       properties:
         auditLogKey:
           allOf:

--- a/zeebe/gateway-protocol/src/main/proto/v2/authentication.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/authentication.yaml
@@ -31,6 +31,7 @@ components:
     CamundaUserResult:
       type: object
       required:
+        - authorizedComponents
         - tenants
         - groups
         - roles

--- a/zeebe/gateway-protocol/src/main/proto/v2/authorizations.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/authorizations.yaml
@@ -301,6 +301,8 @@ components:
             - $ref: '#/components/schemas/ResourceTypeEnum'
 
     AuthorizationResult:
+      required:
+        - permissionTypes
       type: object
       properties:
         ownerId:

--- a/zeebe/gateway-protocol/src/main/proto/v2/batch-operations.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/batch-operations.yaml
@@ -436,6 +436,8 @@ components:
 
     BatchOperationItemResponse:
       type: object
+      required:
+        - rootProcessInstanceKey
       properties:
         operationType:
           $ref: '#/components/schemas/BatchOperationTypeEnum'

--- a/zeebe/gateway-protocol/src/main/proto/v2/batch-operations.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/batch-operations.yaml
@@ -305,6 +305,8 @@ components:
             $ref: '#/components/schemas/BatchOperationResponse'
 
     BatchOperationResponse:
+      required:
+        - errors
       type: object
       properties:
         batchOperationKey:

--- a/zeebe/gateway-protocol/src/main/proto/v2/cluster-variables.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/cluster-variables.yaml
@@ -391,7 +391,6 @@ components:
       type: object
       required:
         - name
-        - value
         - scope
       properties:
         name:

--- a/zeebe/gateway-protocol/src/main/proto/v2/decision-definitions.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/decision-definitions.yaml
@@ -406,6 +406,9 @@ components:
             $ref: '#/components/schemas/EvaluatedDecisionResult'
 
     EvaluatedDecisionResult:
+      required: 
+        - evaluatedInputs
+        - matchedRules
       x-semantic-provider:
         - decisionEvaluationInstanceKey
       type: object

--- a/zeebe/gateway-protocol/src/main/proto/v2/decision-instances.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/decision-instances.yaml
@@ -283,6 +283,8 @@ components:
 
     DecisionInstanceResult:
       type: object
+      required:
+        - rootProcessInstanceKey
       properties:
         decisionEvaluationInstanceKey:
           $ref: 'keys.yaml#/components/schemas/DecisionEvaluationInstanceKey'

--- a/zeebe/gateway-protocol/src/main/proto/v2/decision-instances.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/decision-instances.yaml
@@ -396,6 +396,8 @@ components:
           format: int32
 
     MatchedDecisionRuleItem:
+      required: 
+        - evaluatedOutputs
       type: object
       description: A decision rule that matched within this decision evaluation.
       properties:

--- a/zeebe/gateway-protocol/src/main/proto/v2/decision-instances.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/decision-instances.yaml
@@ -374,9 +374,12 @@ components:
       properties:
         inputId:
           type: string
+          description: The identifier of the decision input.
         inputName:
           type: string
+          description: The name of the decision input.
         inputValue:
+          description: The description of the decision input.
           type: string
 
     EvaluatedDecisionOutputItem:
@@ -384,14 +387,19 @@ components:
       description: The evaluated decision outputs.
       properties:
         outputId:
+          description: The ID of the evaluated decison output item.
           type: string
         outputName:
+          description: The name of the of the evaluated decison output item.
           type: string
         outputValue:
+          description: The value of the evaluated decison output item.
           type: string
         ruleId:
+          description: The ID of the matched rule.
           type: string
         ruleIndex:
+          description: The index of the matched rule.
           type: integer
           format: int32
 

--- a/zeebe/gateway-protocol/src/main/proto/v2/deployments.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/deployments.yaml
@@ -199,15 +199,30 @@ components:
       type: object
       properties:
         processDefinition:
-          $ref: '#/components/schemas/DeploymentProcessResult'
+          nullable: true
+          description: Deployed process.
+          allOf: 
+            - $ref: '#/components/schemas/DeploymentProcessResult'
         decisionDefinition:
-          $ref: '#/components/schemas/DeploymentDecisionResult'
+          nullable: true
+          description: Deployed decision.
+          allOf: 
+            - $ref: '#/components/schemas/DeploymentDecisionResult'
         decisionRequirements:
-          $ref: '#/components/schemas/DeploymentDecisionRequirementsResult'
+          nullable: true
+          description: Deployed decision requirement definition.
+          allOf: 
+            - $ref: '#/components/schemas/DeploymentDecisionRequirementsResult'
         form:
-          $ref: '#/components/schemas/DeploymentFormResult'
+          nullable: true
+          description: Deployed form.
+          allOf: 
+            - $ref: '#/components/schemas/DeploymentFormResult'
         resource:
-          $ref: '#/components/schemas/DeploymentResourceResult'
+          nullable: true
+          description: Deployed resource.
+          allOf: 
+            - $ref: '#/components/schemas/DeploymentResourceResult'
 
     DeploymentProcessResult:
       description: A deployed process.
@@ -290,13 +305,17 @@ components:
         - decisionRequirementsKey
       properties:
         decisionRequirementsId:
+          description: The id of the deployed decision requirements.
           type: string
         decisionRequirementsName:
+          description: The name of the deployed decision requirements.
           type: string
         version:
           type: integer
           format: int32
+          description: The version of the deployed decision requirements.
         resourceName:
+          description: The name of the resource.
           type: string
         tenantId:
           description: The tenant ID of the deployed decision requirements.
@@ -321,9 +340,11 @@ components:
             The form ID, as parsed during deployment, together with the version forms a
             unique identifier for a specific form.
         version:
+          description: The version of the deployed form.
           type: integer
           format: int32
         resourceName:
+          description: The name of the resource.
           type: string
         tenantId:
           $ref: 'identifiers.yaml#/components/schemas/TenantId'
@@ -339,10 +360,13 @@ components:
       type: object
       properties:
         resourceId:
+          description: The resource id of the deployed resource.
           type: string
         resourceName:
+          description: The name of the deployed resource.
           type: string
         version:
+          description: The description of the deployed resource.
           type: integer
           format: int32
         tenantId:

--- a/zeebe/gateway-protocol/src/main/proto/v2/documents.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/documents.yaml
@@ -328,6 +328,8 @@ components:
                 $ref: '#/components/schemas/DocumentReference'
 
     DocumentMetadata:
+      required: 
+        - customProperties
       description: Information about the document.
       type: object
       properties:

--- a/zeebe/gateway-protocol/src/main/proto/v2/element-instances.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/element-instances.yaml
@@ -398,6 +398,7 @@ components:
         - elementInstanceKey
         - processInstanceKey
         - processDefinitionKey
+        - rootProcessInstanceKey
       properties:
         processDefinitionId:
           description: The process definition ID associated to this element instance.

--- a/zeebe/gateway-protocol/src/main/proto/v2/incidents.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/incidents.yaml
@@ -404,6 +404,7 @@ components:
             - $ref: 'identifiers.yaml#/components/schemas/ProcessDefinitionId'
           description: The process definition ID associated to this incident.
         errorType:
+          description: The type of the incident error.
           allOf:
             - $ref: "#/components/schemas/IncidentErrorTypeEnum"
         errorMessage:
@@ -414,9 +415,11 @@ components:
             - $ref: 'identifiers.yaml#/components/schemas/ElementId'
           description: The element ID associated to this incident.
         creationTime:
+          description: The creation time of the incident.
           type: string
           format: date-time
         state:
+          description: The incident state.
           allOf:
             - $ref: "#/components/schemas/IncidentStateEnum"
         tenantId:

--- a/zeebe/gateway-protocol/src/main/proto/v2/incidents.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/incidents.yaml
@@ -396,6 +396,8 @@ components:
 
     IncidentResult:
       type: object
+      required:
+        - rootProcessInstanceKey
       properties:
         processDefinitionId:
           allOf:

--- a/zeebe/gateway-protocol/src/main/proto/v2/job-metrics.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/job-metrics.yaml
@@ -133,6 +133,7 @@ components:
         filter:
           $ref: '#/components/schemas/JobTypeStatisticsFilter'
         page:
+          description: Search cursor pagination.
           allOf:
             - $ref: 'search-models.yaml#/components/schemas/CursorForwardPagination'
 

--- a/zeebe/gateway-protocol/src/main/proto/v2/jobs.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/jobs.yaml
@@ -347,6 +347,7 @@ components:
         - elementInstanceKey
         - kind
         - listenerEventType
+        - rootProcessInstanceKey
       properties:
         type:
           description: The type of the job (should match what was requested).
@@ -647,6 +648,7 @@ components:
         - tenantId
         - type
         - worker
+        - rootProcessInstanceKey
       properties:
         customHeaders:
           description: A set of custom headers defined during modelling.

--- a/zeebe/gateway-protocol/src/main/proto/v2/jobs.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/jobs.yaml
@@ -407,6 +407,7 @@ components:
             - $ref: 'keys.yaml#/components/schemas/ProcessDefinitionKey'
           description: The key of the job's process definition.
         elementInstanceKey:
+          description: The element instance key of the task.
           allOf:
             - $ref: 'keys.yaml#/components/schemas/ElementInstanceKey'
         kind:

--- a/zeebe/gateway-protocol/src/main/proto/v2/jobs.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/jobs.yaml
@@ -348,6 +348,7 @@ components:
         - kind
         - listenerEventType
         - rootProcessInstanceKey
+        - tags
       properties:
         type:
           description: The type of the job (should match what was requested).
@@ -426,6 +427,10 @@ components:
             - $ref: 'keys.yaml#/components/schemas/ProcessInstanceKey'
 
     UserTaskProperties:
+      required: 
+        - candidateGroups
+        - candidateUsers 
+        - changedAttributes
       type: object
       description: Contains properties of a user task.
       properties:

--- a/zeebe/gateway-protocol/src/main/proto/v2/licenses.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/licenses.yaml
@@ -30,6 +30,7 @@ components:
         - validLicense
         - licenseType
         - isCommercial
+        - expiresAt
       properties:
         validLicense:
           description: True if the Camunda license is valid, false if otherwise
@@ -43,7 +44,7 @@ components:
           description: Will be false when a license contains a non-commerical=true property
           type: boolean
         expiresAt:
+          nullable: true
           description: The date when the Camunda license expires
           type: string
           format: date-time
-          nullable: true

--- a/zeebe/gateway-protocol/src/main/proto/v2/messages.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/messages.yaml
@@ -240,6 +240,8 @@ components:
 
     MessageSubscriptionResult:
       type: object
+      required:
+        - rootProcessInstanceKey
       properties:
         messageSubscriptionKey:
           allOf:
@@ -453,6 +455,7 @@ components:
         - processInstanceKey
         - subscriptionKey
         - tenantId
+        - rootProcessInstanceKey
 
     CorrelatedMessageSubscriptionSearchQuery:
       type: object

--- a/zeebe/gateway-protocol/src/main/proto/v2/process-definitions.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/process-definitions.yaml
@@ -461,6 +461,7 @@ components:
       type: object
       properties:
         page:
+          description: Search cursor pagination.
           allOf:
             - $ref: "search-models.yaml#/components/schemas/CursorForwardPagination"
         filter:
@@ -507,6 +508,7 @@ components:
       type: object
       properties:
         page:
+          description: Search cursor pagination.
           allOf:
             - $ref: 'search-models.yaml#/components/schemas/OffsetPagination'
         sort:

--- a/zeebe/gateway-protocol/src/main/proto/v2/process-instances.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/process-instances.yaml
@@ -1189,6 +1189,7 @@ components:
         - tenantId
         - processInstanceKey
         - processDefinitionKey
+        - rootProcessInstanceKey
       properties:
         processDefinitionId:
           $ref: 'identifiers.yaml#/components/schemas/ProcessDefinitionId'
@@ -1289,6 +1290,8 @@ components:
     ProcessInstanceSequenceFlowResult:
       description: Process instance sequence flow result.
       type: object
+      required:
+        - rootProcessInstanceKey
       properties:
         sequenceFlowId:
           type: string

--- a/zeebe/gateway-protocol/src/main/proto/v2/process-instances.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/process-instances.yaml
@@ -1190,6 +1190,8 @@ components:
         - tenantId
         - processInstanceKey
         - processDefinitionKey
+        - parentProcessInstanceKey
+        - parentElementInstanceKey
         - rootProcessInstanceKey
         - tags
       properties:
@@ -1201,15 +1203,18 @@ components:
         processDefinitionVersion:
           type: integer
           format: int32
+          description: The process definition version.
         processDefinitionVersionTag:
           type: string
           description: The process definition version tag.
         startDate:
           type: string
           format: date-time
+          description: The start time of the process instance.
         endDate:
           type: string
           format: date-time
+          description: The completion or termination time of the process instance.
         state:
           $ref: '#/components/schemas/ProcessInstanceStateEnum'
         hasIncident:

--- a/zeebe/gateway-protocol/src/main/proto/v2/process-instances.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/process-instances.yaml
@@ -902,6 +902,7 @@ components:
         - tenantId
         - variables
         - processInstanceKey
+        - tags
       type: object
       properties:
         processDefinitionId:
@@ -1190,6 +1191,7 @@ components:
         - processInstanceKey
         - processDefinitionKey
         - rootProcessInstanceKey
+        - tags
       properties:
         processDefinitionId:
           $ref: 'identifiers.yaml#/components/schemas/ProcessDefinitionId'

--- a/zeebe/gateway-protocol/src/main/proto/v2/system.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/system.yaml
@@ -83,6 +83,8 @@ paths:
 components:
   schemas:
     UsageMetricsResponse:
+      required: 
+        - tenants
       type: object
       allOf:
         - $ref: '#/components/schemas/UsageMetricsResponseItem'

--- a/zeebe/gateway-protocol/src/main/proto/v2/user-tasks.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/user-tasks.yaml
@@ -497,6 +497,10 @@ components:
       type: object
       required:
         - rootProcessInstanceKey
+        - candidateGroups
+        - candidateUsers
+        - customHeaders 
+        - tags
       properties:
         name:
           type: string

--- a/zeebe/gateway-protocol/src/main/proto/v2/user-tasks.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/user-tasks.yaml
@@ -456,10 +456,12 @@ components:
           description: The user task due date.
         processInstanceVariables:
           type: array
+          description: The variables of the process instance.
           items:
             $ref: 'variables.yaml#/components/schemas/VariableValueFilterProperty'
         localVariables:
           type: array
+          description: The local variables of the user task.
           items:
             $ref: 'variables.yaml#/components/schemas/VariableValueFilterProperty'
         userTaskKey:

--- a/zeebe/gateway-protocol/src/main/proto/v2/user-tasks.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/user-tasks.yaml
@@ -495,6 +495,8 @@ components:
 
     UserTaskResult:
       type: object
+      required:
+        - rootProcessInstanceKey
       properties:
         name:
           type: string

--- a/zeebe/gateway-protocol/src/main/proto/v2/variables.yaml
+++ b/zeebe/gateway-protocol/src/main/proto/v2/variables.yaml
@@ -203,6 +203,8 @@ components:
     VariableResultBase:
       description: Variable response item.
       type: object
+      required:
+        - rootProcessInstanceKey
       properties:
         name:
           description: Name of this variable.

